### ISSUE TITLE
[cd] should work with $CDPATH

### ISSUE THREAD

**Issue by SeparateRecords** [withfig/autocomplete#1200]
### Sanity checks

- [X] My issue relates to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If your issue is more general, please create your issue here: [withfig/fig](https://github.com/withfig/fig/issues/new?assignees=&labels=NEED_TO_LABEL&template=main_issue_template.yml)
- [X] I have searched [github.com/withfig/autocomplete/issues](https://github.com/withfig/autocomplete/issues?q=) and there are no duplicates of my issue
- [ ] I would like to work on this.

### What CLI tool does this relate to?

cd

### Which statement makes the most sense?

Something else

### Issue Details

ZSH uses a variable, $CDPATH, to control the search order for `cd`. If empty, it will be `.`. You can add directories to this variable as a colon-separated list.

This is blocked on:
https://github.com/withfig/fig/issues/741
(or https://github.com/withfig/fig/issues/620)

**Comment by SeparateRecords** [comment]
This is now possible but the change actually has to be implemented in the autocomplete engine itself
